packages/theming/src/ThemeProvider/ThemeProvider.js

```diff
@@ -8,13 +8,14 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import { ThemeProvider as StyledThemeProvider } from 'styled-components';
+import merge from 'merge-anything';
+import { default as defaultTheme } from '../theme';
 
 const ThemeProvider = props => {
```

Comment:
From our offline discussion about RTL not being propagated...

For us to use `defaultProps` properly we should move `rtl` and `document` into the `theme` object. This is a breaking change, but makes more sense having rtl as a truly themable value.

Reply:
This would also allow us to just directly export `ThemeProvider` from `styled-components`.